### PR TITLE
Implementation of linearized_directed constructor

### PR DESCRIPTION
**Pull request by d-schindler**
This PR is to provide linearized MS for directed networks as discussed in #69

### CODE REVIEW

**Comment by d-schindler** [comment]
We need to set $\alpha=1$ (no teleportation) as a default for large graphs because otherwise the quality functions is a dense matrix, which leads to memory errors.

**Comment by d-schindler** [comment]
We should have `constructors_kwargs` as an argument in `run()` such that one can change teleportation conveniently.

**Comment by d-schindler** [comment]
@arnaudon , I think this is ready to merge after your improvements. could you e.g. provide a kwarg in run to set the teleportation parameter alpha?

**Comment by arnaudon** [comment]
Here you are!

**Review by arnaudon — APPROVED** [review]

**Comment by arnaudon** [comment]
Let me know if it's ok, and we can merge!

**Comment by d-schindler** [comment]
Do you keep the print statements on purpose? Also the `print("alpha1")`?

File: src/pygenstability/constructors.py
Comment (arnaudon) [inline src/pygenstability/constructors.py:332]:
```suggestion
```

File: src/pygenstability/constructors.py
Comment (arnaudon) [inline src/pygenstability/constructors.py:344]:
```suggestion
```

**Comment by arnaudon** [comment]
Oups, I just removed them!

**Comment by arnaudon** [comment]
is it good to merge like that @d-schindler ?

**Comment by d-schindler** [comment]
Yes, I think it's fine now!

**Comment by d-schindler** [comment]
Before you merge, could you delete this from benchmarking.py, it's deprecated: `with_spectral_decomp=True`

**Pull request merged**
